### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ocefpaf**
@khallock maybe this will help with https://github.com/conda-forge/ncl-feedstock/issues/2

@mingwandroid could you take a look at the Windows build? I have no idea what I am doing there and green at first try is always weird.

@ccordoba12 do you think we need to pin `pixman`? What do you think about his update? Also, can you take a quick look at https://github.com/conda-forge/ncl-feedstock/issues/2 :wink:

### CODE REVIEW

**Comment by mingwandroid** [comment]
@ocefpaf do you know that `m2w64-binutils` is needed here? It likely isn't.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  BINSTAR_TOKEN:
+    # The BINSTAR_TOKEN secure variable. This is defined canonically in conda-forge.yml.
+    secure: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
+
+  matrix:
+    - TARGET_ARCH: x86
+      CONDA_NPY: 110
+      CONDA_PY: 27
+      CONDA_INSTALL_LOCN: C:\\Miniconda
```
Comment (jakirkham) [inline appveyor.yml:24]: This is weird. How did we get a NumPy matrix? 😕

**Comment by ocefpaf** [comment]
> @ocefpaf do you know that `m2w64-binutils` is needed here? It likely isn't.

Nope. Testing it now...

File: appveyor.yml (hunk 1 above)
Comment (jakirkham) [reply appveyor.yml:24]: I have tried to reproduce this several ways and am unable to. If you have a reproducing case, could you please share, @ocefpaf ?

**Comment by jakirkham** [comment]
So the problem now is we are seeing long start up times (~10mins) on Python 2.7 builds with very disconcerting error messages. Could you please try re-rendering with the latest `conda-smithy` again? If you are able to reproduce an issue like with NumPy, I would be very curious to know more details so that we can fix it. 😄

xref: https://github.com/conda-forge/conda-smithy/issues/351

**Comment by ocefpaf** [comment]
> @ocefpaf do you know that m2w64-binutils is needed here? It likely isn't.

Everything seems fine without it! Thanks @mingwandroid!

File: recipe/build.sh [hunk 2]
```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
-./configure --prefix=$PREFIX
-make -j
+
+OPTS=""
+if [[ $(uname) == Darwin ]]; then
+OPTS="--disable-openmp"
```
Comment (ccordoba12) [inline recipe/build.sh]: Please add some indentation here

**Comment by ccordoba12** [comment]
@ocefpaf, sorry, I have no idea about the Mac problem.

**Comment by ocefpaf** [comment]
> @ocefpaf, sorry, I have no idea about the Mac problem.

Thanks for looking into it! It is hard to find people that understands Mac lately :smile:

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      CONDA_PY: 34
+      CONDA_INSTALL_LOCN: C:\\Miniconda3-x64
+
+    - TARGET_ARCH: x86
+      CONDA_PY: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda35
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
```
Comment (ocefpaf) [inline appveyor.yml:44]: Here the issue was the absence of the `posix` package that lead to a unreasonable package in `index`.
See https://github.com/conda-forge/conda-smithy/issues/387#issuecomment-262228001

**Pull request merged**